soc/nordic/common/ironside/Kconfig

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config NRF_IRONSIDE_CALL
+	bool
+	depends on DT_HAS_NORDIC_IPC_IRONSIDE_ENABLED
+	depends on SOC_NRF54H20_IRON
+	select EVENTS
+	select MBOX
+	help
+	  This is selected by features that require support for IRONside calls.
+
+if NRF_IRONSIDE_CALL
+
+config NRF_IRONSIDE_CALL_INIT_PRIORITY
+	int "IRONside calls' initialization priority"
+	default 47
```

Comment:
Should we have a rationale here? Even if the rationale is that its a random value and we don't have a rationale.

Reply:
Added.